packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -1209,9 +1221,74 @@ mixin WidgetsBinding on BindingBase, ServicesBinding, SchedulerBinding, GestureB
 ///    ensure the widget, element, and render trees are all built.
 void runApp(Widget app) {
   final WidgetsBinding binding = WidgetsFlutterBinding.ensureInitialized();
-  assert(binding.debugCheckZone('runApp'));
+  _runWidget(binding.wrapWithDefaultView(app), binding, 'runApp');
+}
+
+/// Inflate the given widget and bootstrap the widget tree.
+///
+// TODO(goderbauer): Update the paragraph bellow to include the Window widget once that exists.
+/// Unlike [runApp], this method does not define a [FlutterView] into which the
+/// provided `app` widget is rendered into. It is up to the caller to include at
+/// least one [View] widget in the provided `app` widget that will bootstrap a
+/// render tree and define the [FlutterView] into which the provided widget is
+/// rendered. Failure to include a [View] widget as an ancestor to all
```

Comment:
One might misread this as "Only a single view can be the ancestor to all render object widgets". 

What do you think of something like "[RenderObjectWidget]s without an ancestor [View] widget will result in an exception"?

Reply:
That phrasing is more clear. Updated.